infrastructure/doc_processing.py

```diff
@@ -82,13 +83,31 @@ def generate_processing_doc():
                 info = param.tooltip_3liz
             else:
                 info = ''
+
+            option = ''
+            if param.defaultValue():
+                option += 'Default: {' + str(param.defaultValue()) + '} <br> '
+
+            if isinstance(param, QgsProcessingParameterNumber):
+                nameType = ''
```

Comment:
You can remove this line. In my IDE, I would have a warning saying this variable is not used, which is true.

Reply:
The empty line ?